code-of-conduct/code-of-conduct-en.md

```diff
@@ -71,8 +71,8 @@ In the event of a non-compliance with the principles in the Code of Conduct, the
 
 - On the first occurrence, the moderator will send a public or private warning, and the content may be modified or deleted;
 - According to the situation, the account of the person may be temporarily suspended;
-- For certain cases, such as extreme behaviours that affect the community, threats, acts of violence, a ban of the community may be applied as well as a prohibition to participate in an event;
-- In case of participation in an event, a member of the organization will contact the alleged perpetrator directly to explain the facts alleged and to inform him of the actions that may be taken against him;
+- For certain extreme behaviours, such as threats or acts of violence, the offender may be banned from the community and be denied the right to participate in the community events;
+- In case of participation in an event, a member of the organization will contact the alleged perpetrator directly to explain the facts alleged and to inform him/her of the actions that may be taken against him/her;
```

Comment:
```suggestion
- In case of participation in an event, a member of the organization will contact the alleged perpetrator(s) directly to explain the facts alleged and to inform them of the actions that may be taken against them;
```